transcendence-app/src/shared/db/models.ts

```diff
@@ -0,0 +1,20 @@
+export enum table {
+  USERS = 'users',
+}
+
+export type ID = string | number;
+
+export interface BaseEntity {
+  id?: string;
+}
+
+export interface Query {
+  text: string;
+  values: Array<string | null>;
+}
+
+export interface MappedQuery {
+  cols: Array<string>;
```

Comment:
There is no difference at all as you say! you can use one or the other. In this case it's just It's more readable to have that generic notation here because next lines have mixed array types (`Array<string | null>`) and it would be weird to mix notations inside the same interface